packages/jaeger-ui-components/src/selectors/trace.ts

```diff
@@ -165,42 +167,50 @@ export const DURATION_FORMATTERS = {
   s: formatSecondTime,
 };
 
-const getDurationFormatterForTrace = createSelector(getTraceDuration, (totalDuration) =>
-  totalDuration >= ONE_SECOND ? DURATION_FORMATTERS.s : DURATION_FORMATTERS.ms
+const getDurationFormatterForTrace = createSelector(getTraceDuration, (totalDuration: number) =>
+  totalDuration! >= ONE_SECOND ? DURATION_FORMATTERS.s : DURATION_FORMATTERS.ms
```

Comment:
Does not seem like we need the `!` here.

Reply:
https://github.com/grafana/grafana/pull/58006/commits/2cde54ce1d190ab77072fd06a0b58ccabcf64ffe